libkernelflinger/trusty_common.c

```diff
@@ -44,6 +44,138 @@
 #include "gpt.h"
 #include "efilinux.h"
 
+#define AVB_COMPILATION
```

Comment:
It is not used

Reply:
AVB_COMPILATION is not defined for libkernelflinger,  so it must be defined here